### PR TITLE
NCL-5099 Fix for publishing when module has no java component

### PR DESCRIPTION
**Pull request by TomasHofman**
Another for openjfx.

### CODE REVIEW

File: manipulation/src/main/java/org/jboss/gm/manipulation/actions/PublishingArtifactsAction.java [hunk 1]
```diff
@@ -28,22 +29,27 @@
...
+                                    mavenPublication.from(javaComponent);
+                                }
+                            });
+                        }
+                    });
+                }
+            });
+        } else {
+            project.getLogger().warn("No java component found for project {}, no artifact to publish.",
+                    project.getName());
```
Comment (rnc) [inline manipulation/src/main/java/org/jboss/gm/manipulation/actions/PublishingArtifactsAction.java]: Does this mean we don't publish then ? i.e. the build will complete but nothing will get uploaded?

Comment (TomasHofman) [reply manipulation/src/main/java/org/jboss/gm/manipulation/actions/PublishingArtifactsAction.java]: For given module, yes. I think root module would be often time the case.

Comment (rnc) [reply manipulation/src/main/java/org/jboss/gm/manipulation/actions/PublishingArtifactsAction.java]: @TomasHofman Sorry didn't quite understand your last comment? I suppose the problem is at this point we know the build might fail to deploy, but failing in the alignment stage doesn't seem right either.

Potentially switch to error logging to try and highlight it better?

Also please note https://github.com/project-ncl/gradle-manipulator/blob/master/CONTRIBUTING.md#code-recommendations - any logging **must** use GMLogger now.

Comment (rnc) [reply manipulation/src/main/java/org/jboss/gm/manipulation/actions/PublishingArtifactsAction.java]: Discussed offline and agreed to just change logger and nothing else

**Comment by TomasHofman** [comment]
PR updated.

**Pull request merged**